tools/kbcheck/src/data.rs

```diff
@@ -29,13 +29,21 @@ pub fn read_json(path: &Path) -> Result<serde_json::Value, DataError> {
     Ok(serde_json::from_reader(reader)?)
 }
 
+fn is_resolved(entry: &DirEntry) -> bool {
+    entry.path()
+         .to_str()
+         .map(|s| s == "data/resolved")
+         .unwrap_or(false)
+}
+
 /// Iterator over DirEntry values corresponding to knowledge base data
 /// files
 pub fn iter_data_files(root_path: &Path) -> impl Iterator<Item = Result<DirEntry, DataError>> {
     let data_path = root_path.to_owned().join("data");
 
     WalkDir::new(&data_path)
         .into_iter()
+        .filter_entry(|e| !is_resolved(e))
         .filter(|maybe_entry| {
```

Comment:
not sure if using an additional filter here is the best way (sorry, not familiar with rust :/ )

Reply:
This now would also exclude resolved entries from validation, not just from scoring. I think it's a not-bad idea to keep validating the resolved items, but not score them, so we'd probably need a `ignore_resolved` argument here or something. @jgraham, wdyt?

Reply:
Yes, I was planning to add a `include_resolved: bool` parameter. I'll fix this up quickly so we can exclude resolved bugs from the report.

Reply:
Thanks!!